app/controllers/api/v1/api_controller.rb

```diff
@@ -18,9 +18,9 @@ def render_error_response(error)
       end
 
       def render_not_found_response
-        render json: { error: I18n.t(
-          'api.error.invalid_request.content_not_found'
-        ) }, status: :not_found
+        render json: { error: I18n
+          .t('api.error.invalid_request.content_not_found') },
+               status: :not_found
```

Comment:
Why this change?

Reply:
not sure, probably trying to make it look prettier 🤔

Reply:
haha ok, I think it is a weird indentation, but it's up to you 😄

Reply:
fair enough, what would not be weird @gonzaM ?
Anyway ill probably be weird until i finally make friends with rubo🤖